test/JDBC/expected/fts-contains-vu-verify.out

```diff
@@ -2636,6 +2636,391 @@ int#!#text
 ~~END~~
 
 
+select t.name from test_special_char_t t where contains(t.name, '"one two"');
```

Comment:
Please add tests for column alias, cross database queries
Add tests where we alias with table name, schema.table name and cross db table name